content/en/tracing/trace_pipeline/trace_retention.md

```diff
@@ -57,28 +57,25 @@ Last Updated
 Enabled toggle
 : Allows filters to be turned on and off.
 
-The order of the retention filter list changes indexing behavior. If a span matches a retention filter early in the list, the span is either kept or dropped. Any matching retention filter lower on the list does not catch the already-processed span.
+**Note**: The order of the retention filter list changes indexing behavior. If a span matches a retention filter early in the list, the span is either kept or dropped. Any matching retention filter lower on the list does not catch the already-processed span.
 
 The `Spans Indexed` column for each retention filter is powered by the `datadog.estimated_usage.apm.indexed_spans` metric, which you can use to track your indexed span usage. For more information, read [Usage Metrics][2], or see the [dashboard][4] available in your account.
 
 <div class="alert alert-info"><strong>Note</strong>: Retention filters do not affect what traces are collected by the Agent and sent to Datadog ("ingested"). The only way to change how much tracing data is ingested is through <a href="/tracing/trace_ingestion/mechanisms">ingestion controls</a>.</div>
 
 ### Datadog intelligent retention filter
 
-The Datadog intelligent retention filter is always active for your services, and it keeps a proportion of traces to help you monitor the health of your applications. 
+The Datadog intelligent retention filter is always active for your services, and it keeps a representative selection of traces without requiring you to create dozens of custom retention filters.
 
 It scans through the **service entry spans** and retains for 30 days:
 
- - A representative selection of errors, ensuring error diversity (for example, response code 400s, 500s).
- - High latency in the `p75`, `p90`, and `p95` percentiles.
- - All resources with any traffic that have associated traces in the past for any time window selection.
- - True maximum duration trace for each time window.
+- At least one span (and the associated trace) for each combination of environment, service, operation and resource every 15 minutes at most to ensure to always find example traces in [service][9] and [resource][10] pages, even for low traffic endpoint.
```

Comment:
should we add something here to remind the reader that we can only retain spans that are sent to our backend (ingested)?